llvm/lib/Target/BPF/BPFISelLowering.cpp

```diff
@@ -703,6 +714,20 @@ SDValue BPFTargetLowering::LowerSELECT_CC(SDValue Op, SelectionDAG &DAG) const {
   return DAG.getNode(BPFISD::SELECT_CC, DL, VTs, Ops);
 }
 
+SDValue BPFTargetLowering::LowerATOMIC_LOAD_STORE(SDValue Op,
+                                                  SelectionDAG &DAG) const {
+  SDNode *N = Op.getNode();
+  SDLoc DL(N);
+
+  if (cast<AtomicSDNode>(N)->getMergedOrdering() ==
+      AtomicOrdering::SequentiallyConsistent)
+    fail(
+        DL, DAG,
+        "sequentially consistent (seq_cst) atomic load/store is not supported");
```

Comment:
You can break the string literal to avoid the wrapping
```suggestion
        "sequentially consistent (seq_cst) "
        "atomic load/store is not supported");
```

Reply:
Got it, thanks!